src/analyses/apron/relationPriv.apron.ml

```diff
@@ -489,7 +489,11 @@ struct
     in
     let get_mutex_inits = getg V.mutex_inits in
     let get_mutex_inits' = RD.keep_vars get_mutex_inits [g_var] in
-    RD.join get_mutex_global_g get_mutex_inits'
+    if not (RD.mem_var get_mutex_inits' g_var) then
+      (* This is an escaped variable whose value was never side-effected to get_mutex_inits' *)
```

Comment:
Is this not just a workaround for the actual bug? Seems like escapes should also contribute to `mutex_inits`.

`BasePriv.PerMutexPrivBase` and `BasePriv.PerMutexMeetTIDPriv` side-effect escapes to `mutex_inits` at least.

Reply:
Yes, in the base priv this is all not problematic, as it is non-relational. Here, this all becomes fragile, and I think for now we are wise to avoid the set of variables at global unknowns not only shrinking (due to decreasing protection) but also increasing in non-obvious matters due to escapes also being tracked associated with `mutex_inits` which is supposed to abstract the state at program start.  Maybe this is also something for #1380.